src/components/Header.js

```diff
@@ -18,7 +18,7 @@ function Header(props) {
     return (
         <div className='nav'>
             <h1>Fitness Tracker</h1>
-            {(loggedIn === true)
+            {(getToken())
```

Comment:
This was just a temporary fix on my part to get the logout button to display for me. feel free to change as you wish.